ui/app/mainui/Popups.qml

```diff
@@ -675,7 +683,38 @@ QtObject {
             ImportControlNodePopup {
                 onClosed: destroy()
             }
-        }
+        },
+
+        Component {
+            id: editSharedAddressesPopupComponent
+            SharedAddressesPopup {
+                id: editSharedAddressesPopup
 
+                property string communityId
+
+                readonly property var chatStore: ChatStore.RootStore {
+                    contactsStore: root.rootStore.contactStore
+                    chatCommunitySectionModule: {
+                        root.rootStore.mainModuleInst.prepareCommunitySectionModuleForCommunityId(editSharedAddressesPopup.communityId)
+                        return root.rootStore.mainModuleInst.getCommunitySectionModule()
+                    }
+                }
+
+                communityName: chatStore.sectionDetails.name
+                communityIcon: chatStore.sectionDetails.image
+                // FIXME get these from the community settings (from the initial "join" call)
+                //selectedSharedAddresses: [???]
+                //selectedAirdropAddress: "???"
```

Comment:
This is supposed to be done in the subsequent issue or in this one?

Reply:
I logged a separate issue for this: https://github.com/status-im/status-desktop/issues/11822

Reply:
Good, that's what I thought, I just wanted to be sure.

Reply:
Also will need https://github.com/status-im/status-desktop/issues/11480, ie something that triggers the rebuild of the permissions model based on what addresses are checked